jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+          example: MSK
+          maxLength: 100
+        codeListName:
+          type: string
+          x-stoplight:
+            id: sqwp8c5h95mi8
+          description: 'The name of the list, provided by the responsible agency'
+          example: LCL
+          maxLength: 100
+    Vessel:
+      title: Vessel
+      x-stoplight:
+        id: c5swjeesh773k
+      type: object
+      description: 'A floating, sea going structure (mother vessels and feeder vessels) with either an internal or external mode of propulsion designed for the transport of cargo and/or passengers. Ocean vessels are uniquely identified by an IMO number consisting of 7 digits.'
+      required:
+        - vesselIMONumber
+      properties:
+        vesselIMONumber:
+          type: string
+          x-stoplight:
+            id: j6xz8w07lptx6
+          description: 'The unique reference for a registered Vessel. The reference is the International Maritime Organisation (IMO) number, also sometimes known as the Lloyd''s register code, which does not change during the lifetime of the vessel.'
+          example: '9321483'
+          maxLength: 7
+        name:
+          type: string
+          x-stoplight:
+            id: ru335qkry8k93
+          description: The name of the Vessel given by the Vessel Operator and registered with IMO.
+          maxLength: 35
+          example: King of the Seas
+        lengthOverall:
+          type: number
+          x-stoplight:
+            id: ctseu7r51zzzh
+          format: float
+          description: "The maximum length of a ship's hull measured parallel to the waterline (Length OverAll).\r\n\r\nIf the length is specified in feet (`FOT`) then the decimal part should be concidered as a fraction of a foot and not as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6\")."
+          example: 245.45
+        width:
+          type: number
+          x-stoplight:
+            id: 3dlryy1aenm62
+          format: float
+          example: 37.33
+          description: "Overall width of the ship measured at the widest point of the nominal waterline.\r\n\r\nIf the width is specified in feet (`FOT`) then the decimal part should be concidered as a fraction of a foot and not as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6\")."
+        callSign:
+          type: string
+          x-stoplight:
+            id: xut9lzy2lig22
+          description: 'A unique alphanumeric identity that belongs to the vessel and is assigned by the International Telecommunication Union (ITU). It consists of a threeletter alphanumeric prefix that indicates nationality, followed by one to four characters to identify the individual vessel. For instance, vessels registered under Denmark are assigned the prefix ranges 5PA-5QZ, OUAOZZ, and XPA-XPZ. The Call Sign changes whenever a vessel changes its flag.'
+          example: NCVV
+          maxLength: 10
+        type:
```

Comment:
In the current version this is an Enum

Reply:
vesselCallSignNumber?
<img width="518" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/138600841/4b45b3de-dd99-4d63-89cc-f8983530f354">